README.md

```diff
@@ -5,7 +5,7 @@
 [![Coverage](https://next.sonarqube.com/sonarqube/api/badges/measure?key=sonarlint-visualstudio&metric=coverage)](https://next.sonarqube.com/sonarqube/component_measures/domain/Coverage?id=sonar-scanner-msbuild)
 
 
-The SonarQube Scanner for MSBuild is the recommended way to launch a SonarQube analysis for projects/solutions using
+The Sonar Scanner for MSBuild is the recommended way to launch SonarQube and SonarCloud analysis for projects/solutions using
```

Comment:
[minor] I think it should be `SonarScanner` here too or the other instances should be renamed as `Sonar Scanner`.